get-annotation.sh

```diff
@@ -149,13 +149,14 @@ fi
 token_quoted=$(echo $token_json | jq .token)
 token_raw=$(echo $token_quoted | sed 's/\"//g')
 
-echo $URL
-curl --fail --silent --show-error -H "X-Auth-Token: $token_raw" -X GET $URL | jq
+response=$(curl --fail --silent --show-error -H "X-Auth-Token: $token_raw" -X GET $url)
 curl_result=$?
 
 if [ $curl_result -ne 0 ]; then
-  echo "Failed to delete annotation (curl exit code: $curl_result)."
+  echo "Failed to retrieve annotation(s) (curl exit code: $curl_result)."
   exit 1
+else
+  echo $response | jq
```

Comment:
@wsmithee wrap `$response` in quotes here to avoid problems with whitespace and other Bashy stuff.

Reply:
Will do.